### ISSUE TITLE
Notes

### ISSUE THREAD

**Issue by matkoniecz** [piebro/OpenStreetMap-Statistics#36]
Notes are also potentially useful!

https://resultmaps.neis-one.org/osm-notes have plenty of interesting things, but visualisation of open note count from

- all notes
- onosm.org
- StreetComplete
- StreetComplete with photo
- all notes except this identified sources

would be interesting.

Would require a new pipeline for notes dump.

**Comment by piebro** [comment]
I agree, I haven't looked into notes yet. I'm not sure if they are save in a changeset, at least the wiki pages doesn't mention notes https://wiki.openstreetmap.org/wiki/Changeset. Do you know of a tag which specifies if a note was added?

**Comment by matkoniecz** [comment]
Notes are in a separate dataset - https://planet.openstreetmap.org/notes/ ("Would require a new pipeline for notes dump." was too cryptic expression of that)

(iD records closed notes during making given edit but it is iD-specific changeset tag)

**Comment by piebro** [comment]
Ah I see. If it has a similar structure it should be totally doable. The `planet-notes-latest.osn` file is an xml and I should be able to handle it quite similar to the changeset file. I think this would be an interesting addition, but I will focus on making the changeset stats more polished first.

**Comment by matkoniecz** [comment]
Vespucci allows to fill it: https://www.openstreetmap.org/changeset/129726146

iD has some mediocre support for it and is barely used

StreetComplete fills it automatically based on GPS location: https://www.openstreetmap.org/changeset/129749863

**Comment by piebro** [comment]
I just did a big refactoring of the code and also added a notes and comment on notes dataset. There are also some statistics about these notes, but they are quite basic, but still cool: https://piebro.github.io/openstreetmap-statistics/stats/10_notes.html

I also tried to make the whole process of adding new plots a lot easier. Websites are self contained Jupyter Notebook now with SQL queries and some helper functions for plotting and tables.

If you are interested you can try to add some more plots about notes. Feel free to ask question if you get stuck or something is missing or does not work.
